docs/specs/meta-clients/web3modal/api.md

```diff
@@ -1,142 +1,248 @@
-  headers: Headers;
-}
+#### Response
 
-type Response = Blob;
+```typescript
+{
+  "count": number,
+  "data": [
+    {
+      "id": string,
+      "ios_schema": string
+    }
+  ]
+}
 ```
 
-### GET `/public/getAssetImage/:asset_id`
+### `GET /v2/android-data`
 
-Returns image for public asset for requested `asset_id`, i.e network logo, token logo or connector logo.
+Returns a list of Android-facing wallet data, including the public Explorer ID and Android App ID.
 
-```ts
-interface Request {
-  headers: Headers;
-}
+#### Response
 
-type Response = Blob;
+```typescript
+{
+  "count": number,
+  "data": [
+    {
+      "id": string,
+      "android_app_id": string
+    }
+  ]
+}
 ```
 
-#### Known Static Asset IDs
+### `GET /v2/wallet-image/{size}/{wallet_id}`
+
+Returns the image logo for the specified wallet ID.
+
+#### Route Parameters
+
+- `wallet_id`: The unique identifier for the wallet
+- `size`: The predefined image size, described as `{width}x{height}` (see [Known Image Sizes](#known-image-sizes))
+
+#### Response
+
+- A binary image file (Blob)
+
+### `GET /v2/asset-image/network/{size}/{asset_id}`
+
+Fetches image for public assets like network logos using well-defined CAIP identifiers.
+
+#### Route Parameters
+
+- `asset_id`: CAIP identifier for the network (e.g., `eip155-1`)
+- `size`: The predefined image size, described as `{width}x{height}` (see [Known Image Sizes](#known-image-sizes))
+
+#### Example API Requests
+
+- Ethereum: `GET /v2/asset-image/network/56x56/eip155-1`
+- Arbitrum: `GET /v2/asset-image/network/56x56/eip155-42161`
+- Avalanche: `GET /v2/asset-image/network/56x56/eip155-43114`
+- Binance Smart Chain: `GET /v2/asset-image/network/56x56/eip155-56`
+- Fantom: `GET /v2/asset-image/network/56x56/eip155-250`
+- Optimism: `GET /v2/asset-image/network/56x56/eip155-10`
+- Polygon: `GET /v2/asset-image/network/56x56/eip155-137`
+- Gnosis: `GET /v2/asset-image/network/56x56/eip155-100`
+- Evmos: `GET /v2/asset-image/network/56x56/eip155-9001`
+- ZkSync: `GET /v2/asset-image/network/56x56/eip155-324`
+- Filecoin: `GET /v2/asset-image/network/56x56/fil-314`
+- IoTeX: `GET /v2/asset-image/network/56x56/eip155-4689`
+- Metis: `GET /v2/asset-image/network/56x56/eip155-1088`
+- Moonbeam: `GET /v2/asset-image/network/56x56/eip155-1284`
+- Moonriver: `GET /v2/asset-image/network/56x56/eip155-1285`
+- Zora: `GET /v2/asset-image/network/56x56/eip155-7777777`
+- Celo: `GET /v2/asset-image/network/56x56/eip155-42220`
+- Base: `GET /v2/asset-image/network/56x56/eip155-8453`
+- Aurora: `GET /v2/asset-image/network/56x56/eip155-1313161554`
+
+#### Response
+
+- A binary image file (Blob)
 
-There are a number of known static asset IDs that can be used to fetch images, e.g. for networks and connectors:
+### `GET /v2/asset-image/connector/{size}/{connector_id}`
+
+Fetches image for connector logos using descriptive names.
+
+#### Route Parameters
+
+- `connector_id`: Descriptive name for the connector (e.g., `coinbase-wallet`)
```

Comment:
are we using well-known names for these e.g. their rdns? else how do we get the list of available connectors? do we need to hardcode it ?

Reply:
Agree with Rocky here we should use the `rdns`

Reply:
Agreed, RDNS makes a lot more sense. Will update 👍

Reply:
@tomiir I just noticed that a number of connectors that are currently being mapped [here in PresetsUtil](https://github.com/WalletConnect/web3modal/blob/main/packages/scaffold-utils/src/PresetsUtil.ts#L10) (e.g. Safe, Ledger) don't have `rdns` values set in Cloud and hence the `/getWallets` response.

Wouldn't this then still necessitate manual mappings in the client either way?

Reply:
What's the resolution here ? @tomiir

Reply:
Coinbase we use `coinbaseWalletSDK` as Chris mentions, but you are correct @bkrem that Ledger, Safe and other wallets that are not 6963 might not have it :| 

I think there's no way around the id list then, unless there's a known global ID we can use instead :( 
We'd still need to somehow fetch the id, I guess we already have it from requesting all wallets before